docs/book/src/topics/eks/pod-networking.md

```diff
@@ -7,6 +7,27 @@ When creating a EKS cluster the Amazon VPC CNI will be used by default for Pod N
 ## Using the VPC CNI Addon
 You can use an explicit version of the Amazon VPC CNI by using the **vpc-cni** EKS addon. See the [addons](./addons.md) documentation for further details of how to use addons.
 
+
+## Increase node pod limit
+You can increase the pod limit per-node as [per the upstream AWS documentation](https://aws.amazon.com/blogs/containers/amazon-vpc-cni-increases-pods-per-node-limits/). You'll need to enable the `vpc-cni` plugin addon on your EKS cluster as well as enable prefix assignment mode through the `ENABLE_PREFIX_DELEGATION` environment variable.
+
+```yaml
+kind: AWSManagedControlPlane
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+metadata:
+  name: "capi-managed-test-control-plane"
+spec:
+  awsNodeEnvVars:
+  - name: ENABLE_PREFIX_DELEGATION
+    value: "true"
+  addons:
```

Comment:
Is there a reason to specify vpc-cni in addons when`disableVPCCNI: false` is already set? Is it to pick a specific version?

Reply:
No particular reason, I just wanted to be as explicit as possible with the example configuration! Happy to change it if you think it's redundant?

Reply:
Make sense. In that case, can we change the version to sth like REPLACE_WITH_VPC_CNI_VERSION instead of a specific version? As this is a doc, I worry that it will get outdated and we will need to update it regularly.